manifests/candlepin.pp

```diff
@@ -72,9 +74,8 @@
   $truststore_password = extlib::cache_data('foreman_cache_data', $truststore_password_file, extlib::random_password(32))
   $keystore_password_path = "${pki_dir}/${keystore_password_file}"
   $truststore_password_path = "${pki_dir}/${truststore_password_file}"
-  $client_req = "${pki_dir}/java-client.req"
-  $client_cert = "${pki_dir}/certs/${java_client_cert_name}.crt"
-  $client_key = "${pki_dir}/private/${java_client_cert_name}.key"
+  $client_key = $certs::foreman::client_cert
+  $client_cert = $certs::foreman::client_key
```

Comment:
The reason to redefine these is it allows us to keep the current wiring code intact -- https://github.com/theforeman/puppet-katello/blob/master/manifests/application.pp#L50-L51